package/MDAnalysis/analysis/legacy/x3dna.py

```diff
@@ -423,7 +423,7 @@ def plot(self, **kwargs):
             ax.set_xlabel(r"Nucleic Acid Number")
             param = self.profiles.values()[0].dtype.names[k]
             if param in ["Shear", "Stretch", "Stagger", "Rise", "Shift", "Slide"]:
-                ax.set_ylabel("{0!s} ($\AA$)".format((param)))
+                ax.set_ylabel(r"{!s} ($\AA$)".format(param))
```

Comment:
Why is this being changed?

Reply:
This line generates a warning about invalid escape sequences. I have used pyupgrade to fix these warnings : https://github.com/asottile/pyupgrade#invalid-escape-sequences

```
find . -iname '*.py' | xargs -P4 -I{} python3.8 -Wall -m py_compile {}   
./testsuite/MDAnalysisTests/lib/test_util.py:1477: SyntaxWarning: "is" with a literal. Did you mean "=="?
  assert myfunc.foo is 0
./testsuite/MDAnalysisTests/lib/test_util.py:1491: SyntaxWarning: "is" with a literal. Did you mean "=="?
  assert x[0] is 1
./testsuite/MDAnalysisTests/lib/test_util.py:1492: SyntaxWarning: "is" with a literal. Did you mean "=="?
  assert myfunc.bar['test'][0] is 1
./testsuite/MDAnalysisTests/lib/test_util.py:1498: SyntaxWarning: "is" with a literal. Did you mean "=="?
  assert myfunc.bar['test'][0] is 2
./testsuite/MDAnalysisTests/lib/test_util.py:1539: SyntaxWarning: "is" with a literal. Did you mean "=="?
  assert x is 0
./testsuite/MDAnalysisTests/lib/test_util.py:1551: SyntaxWarning: "is" with a literal. Did you mean "=="?
  assert x is 0
./testsuite/MDAnalysisTests/lib/test_util.py:1605: SyntaxWarning: "is" with a literal. Did you mean "=="?
  assert x is 0
./package/MDAnalysis/analysis/legacy/x3dna.py:426: DeprecationWarning: invalid escape sequence \A
  ax.set_ylabel("{0!s} ($\AA$)".format((param)))
```